src/sqlancer/questdb/gen/QuestDBExpressionGenerator.java

```diff
@@ -54,6 +55,15 @@ public Node<QuestDBExpression> generateConstant() {
             return QuestDBConstant.createBooleanConstant(Randomly.getBoolean());
         case FLOAT:
             return QuestDBConstant.createFloatConstant(globalState.getRandomly().getDouble());
+        case SYMBOL:
+            StringGenerationStrategy strategy = Randomly.StringGenerationStrategy.ALPHANUMERIC;
+            // Add quotes to form a valid string for QuestDB
+            String str = '\"' + strategy.getString(globalState.getRandomly()) + '\"';
```

Comment:
In SQL, strings are typically in single quotes. Are you sure that QuestDB uses double quotes?